spacewalk/certs-tools/mgr_ssl_cert_setup.py

```diff
@@ -103,39 +101,51 @@ def processCommandline():
     return options
 
 
-def checkOptions(options):
-    if not options.root_ca_file:
+def checkOptions(root_ca_file, server_cert_file, server_key_file, intermediate_ca_file):
+    if not root_ca_file:
         log_error("Root CA is required")
         sys.exit(1)
-    if not os.path.exists(options.root_ca_file):
-        log_error("Root CA: file not found {}".format(options.root_ca_file))
+    if not os.path.exists(root_ca_file):
+        log_error("Root CA: file not found {}".format(root_ca_file))
         sys.exit(1)
 
-    if not options.server_cert_file:
+    if not server_cert_file:
         log_error("Server Certificate is required")
         sys.exit(1)
-    if not os.path.exists(options.server_cert_file):
+    if not os.path.exists(server_cert_file):
         log_error(
-            "Server Certificate: file not found {}".format(options.server_cert_file)
+            "Server Certificate: file not found {}".format(server_cert_file)
         )
         sys.exit(1)
 
-    if not options.server_key_file:
+    if not server_key_file:
         log_error("Server Private Key is required")
         sys.exit(1)
-    if not os.path.exists(options.server_key_file):
+    if not os.path.exists(server_key_file):
         log_error(
-            "Server Private Key: file not found {}".format(options.server_key_file)
+            "Server Private Key: file not found {}".format(server_key_file)
         )
         sys.exit(1)
 
-    for ica in options.intermediate_ca_file:
+    for ica in intermediate_ca_file:
         if not os.path.exists(ica):
             log_error("Intermediate CA: file not found {}".format(ica))
             sys.exit(1)
 
 
-def prepareWorkdir(options, workdir):
+def readAllFiles(root_ca_file, server_cert_file, server_key_file, intermediate_ca_file):
+    allFiles = [root_ca_file, server_cert_file, server_key_file]
+    allFiles.extend(intermediate_ca_file)
+
+    contents = []
+    for file in allFiles:
```

Comment:
It could be that this was true in Python 2, but "file" is not a keyword in Python 3:
```
>>> import keyword
>>> keyword.iskeyword("file")
False
```